ci/ci-release-build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why out of curiosity?

Reply:
https://nixos.wiki/wiki/NixOS#Comparison_with_traditional_Linux_Distributions

Not really relevant here, I can drop that if preferred. I seldomly use `nix` myself, but it's more like a good habit.